CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Module now fully supports the use of `cert_file` parameter.
```

Comment:
```suggestion
- Module now fully supports the use of `cert_file` parameter, which accepts the path to the PEM-encoded
  x509 CA certificate chain for Conjur. Users can supply the module with certificate path in `cert_file` OR
  with certificate contents in `ssl_certificate`.
```
suggestion to be more clear about what `cert_file` is in this message (eg what does this _mean_ to the end user?)